tests/lib/rules/no-extra-parens.js

```diff
@@ -1122,12 +1153,10 @@ ruleTester.run("no-extra-parens", rule, {
                 }
             ]
         },
-
-        // ["all", { enforceForArrowConditionals: false }]
         {
             code: "var a = (b) => ((1 ? 2 : 3))",
             output: "var a = (b) => (1 ? 2 : 3)",
-            options: ["all", { enforceForArrowConditionals: false }],
```

Comment:
This was definitely intentional ([#8439 comment](https://github.com/eslint/eslint/pull/8439#discussion_r115153519)). Unfortunately, other options don't work like this, so it's better to align.